lexicon/conversion.py

```diff
@@ -140,6 +140,69 @@ def run(self):
             lexicon_file.write(ET.tostring(root, "unicode"))
 
 
+class LexiconFromTextFileJob(Job):
+    """
+    Create a bliss lexicon from a regular text file, where each line contains:
+    < WORD > < PHONEME1 > < PHONEME2 > ...
```

Comment:
We could also remove the spaces inside the brackets, I just had to insert them because otherwise nothing would show up in the comment here.